setup/install/providers/aws/aws-ec2.md

```diff
@@ -25,10 +25,12 @@ Use this option to deploy Spinnaker, if you are familar with deployment using [A
 1. Navigate to [Console](https://console.aws.amazon.com/){:target="\_blank"} > CloudFormation and [select](https://docs.aws.amazon.com/awsconsolehelpdocs/latest/gsg/getting-started.html#select-region) your preferred region.
 2. Download [the template](https://d3079gxvs8ayeg.cloudfront.net/templates/managing.yaml) locally to your workstation.
 
-    2.a (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
+    2.a Search for ‘SpinnakerInstanceProfileArn’ and comment out the line.
+    
+    2.b (Optional). Add additional managed account as shown on line 158 in the SpinnakerAssumeRolePolicy section of the downloaded template file.
 3. Creating the CloudFormation Stack
```

Comment:
```suggestion
3. Create the CloudFormation Stack:
```

Reply:
@anooptejt , I know this was already like this before you got in this file, but feel free to improve it while you're in here. :)